src/database/20230622215327_create_table_locomotive.ts

```diff
@@ -0,0 +1,17 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable("locomotive", function (table) {
+    table.increments();
+    table.string("name");
+    table.string("status");
+    table.string("route");
+    table.string("load");
+    table.string("trainDriver");
```

Comment:
O que acha de seguirmos a sugestão da Leka e criarmos o campo driver_id como uma FK para a tabela de Driver que criaremos?
```suggestion
    // TODO- ADD FK AFTER TABLE IS CREATED
    table.integer("driver_id");
```